.dockerignore

```diff
@@ -31,3 +32,5 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+docker
+.deps
```

Comment:
It's very strange that `.deps` also causes cache misses, It took a long time to locate it.